scripts/cy-wrapper.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+if test -z "$BASH_VERSION"; then
+  echo "Please run this script using bash, not sh or any other shell." >&2
+  exit 1
+fi
+
+export CY_API_URL="${CY_API_URL:-https://http-api.cycloid.io}"
+export CY_BINARIES_PATH="${CY_BINARIES_PATH:-$HOME/.cycloid}"
+
+# Ensure we have Cycloid directory created
+if ! [ -d "$CY_BINARIES_PATH" ]; then
+  mkdir -p $CY_BINARIES_PATH
+fi
+
+# If specified in the commandline use it
+if [[ `echo "$@" | grep -E --  "--api-url(=|\s+)([^\]+)"` ]]; then
+  CY_API_URL=$(echo "$@" | sed -r "s/.*--api-url(=|\s+)([^ ]+).*/\2/")
+fi
+
+# Remove trailing /
+CY_API_URL=${CY_API_URL%/}
+
+# Get Cycloid API version
+export CY_VERSION=$(curl -s "${CY_API_URL}/version" | jq -r .data.version)
+
+if [[ -z "$CY_VERSION" ]]; then
+  echo "Error: Unable to get Cycloid API version on ${CY_API_URL}/version"
+  exit 1
+fi
+
+# Download the binary if not present
+export CY_BINARY="${CY_BINARIES_PATH}/${CY_VERSION}"
```

Comment:
We could set an explicit name ? 

```suggestion
export CY_BINARY="${CY_BINARIES_PATH}/cy-${CY_VERSION}"
```